### ISSUE TITLE
wrong and unchangable  font or language settings make darktable unusable under macos catalina 10.15.1

### ISSUE THREAD

**Issue by tlo8640** [darktable-org/darktable#3500]
**Describe the bug**
I have a Macbook Pro 2015 (without touchbar) with english keyboard but set to swiss german keyboard layout and german )swiss german(?) language. I tried installing darktable as dmg image from your homepage or over brew (at least brew does the same). So I got the same result:
When starting darktable it's unusable because I can't read anything (screenshot):
<img width="1440" alt="grafik" src="https://user-images.githubusercontent.com/740897/69275957-bb85af80-0bdd-11ea-88f6-97fd7a0ec195.png">

I tried different language settings in preferences, but it didn't helped.
manually setting the language in darktabkerc to "en" or "de" didn't helped too.
What could be wrong here?

**To Reproduce**
I really tried new installation of darktable more than one time with removing everything what could belong to darktable .config and .cache, but no success.
I tried darktable on the old macbook pro 15"(late 2008 with OSX 10.10.5 swiss german keyboard and keyboard layout) and and got the same result.

**Expected behavior**
Darktable should be readable and usable :-)

**Screenshots**
Is attached

**Platform (please complete the following information):**
 - OS: macos 10.15.1 and osx 10.10.5
 - Version 2.6.3.1

Perhaps it has somethng to do with the installed fonts? on the old laptop there is a huge amount of different fonts installed and the new one got a time machine restore while setting up (but only the user homes) Can I check this somehow, where are the font settings of darktable? Do I have to install additional fonts? How about the themes where can I change them?

**Comment by tlo8640** [comment]
ok, I don't know why I didn't it before opening this issue...
I restored font settings to default (that means I removed all additional installed fonts and reinstalled macos/osx default fonts) using the font collection app in macos.
After finishing this task I gave darktable another try and what can I say, everything was fine!
I could read any text in the user interface and everythings sems to be fine.
Something was wrong with the additional installed fonts.
